src/pages/content-edit/name.js

```diff
@@ -1,5 +1,5 @@
 "use strict";
 
 export default function name(schema, data) {
-    return data.name || "Untitled " + schema.name;
+    return data.name || schema.name && "Untitled " + schema.name || "...";
```

Comment:
Wrap that middle condition in parens please so it's more obviously joined together.